tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -76,6 +76,13 @@ public static function wpSetUpBeforeClass( $factory ) {
 		set_post_thumbnail( self::$story_id, $poster_attachment_id );
 	}
 
+	public function tearDown() {
```

Comment:
Can you explain this / add a comment here.

Reply:
This is just some regular cleanup for these tests. Some tests call `$this->set_permalink_structure( '/%postname%/' );`, so it's important to reset that again; otherwise it will affect other tests in other files.